src/toolkit.c

```diff
@@ -1484,29 +1495,8 @@ static void window_renderBorder( const Window *w )
    /* Position */
    double x = w->x;
    double y = w->y;
-
-   /*
-    * Case fullscreen.
-    */
-   if ( window_isFlag( w, WINDOW_FULLSCREEN ) ) {
```

Comment:
I think it would be nice to have a fullscreen fallback for the land window and such otherwise the tabs look a bit weird, like we had without the rounded tabs. Although I don't think this is the place.